exonum-java-binding/core/rust/src/runtime/java_service_runtime.rs

```diff
@@ -80,26 +80,26 @@ impl JavaServiceRuntime {
     /// Creates a new service instance using the given artifact id.
     ///
     /// Panics if there are errors on Java side.
-    pub fn create_service(&self, artifact_id: &str) -> ServiceProxy {
-        unwrap_jni(self.executor.with_attached(|env| {
-            let artifact_id_obj: JObject = env.new_string(artifact_id)?.into();
-            let service = panic_on_exception(
-                env,
-                env.call_method(
-                    self.service_runtime.as_obj(),
-                    "createService",
-                    CREATE_SERVICE_SIGNATURE,
-                    &[artifact_id_obj.into()],
-                ),
-            )
-            .l()?;
-            let service = env.new_global_ref(service)?;
-            Ok(ServiceProxy::from_global_ref(
-                self.executor.clone(),
-                service,
-            ))
-        }))
-    }
+//    pub fn create_service(&self, artifact_id: &str) -> ServiceProxy {
+//        unwrap_jni(self.executor.with_attached(|env| {
+//            let artifact_id_obj: JObject = env.new_string(artifact_id)?.into();
+//            let service = panic_on_exception(
+//                env,
+//                env.call_method(
+//                    self.service_runtime.as_obj(),
+//                    "createService",
+//                    CREATE_SERVICE_SIGNATURE,
+//                    &[artifact_id_obj.into()],
+//                ),
+//            )
+//            .l()?;
+//            let service = env.new_global_ref(service)?;
+//            Ok(ServiceProxy::from_global_ref(
+//                self.executor.clone(),
+//                service,
+//            ))
+//        }))
+//    }
 
     /// Loads an artifact from the specified location involving verification of the artifact.
```

Comment:
Hasn't this method moved to the runtime and no longer needed? Also, the naming of this class needs to be updated because it conflicts with the runtime.